.drone.yml

```diff
@@ -50,18 +56,94 @@ steps:
       email: lucas@loeffel.io
       folder: npm
       access: public
-      # tag: beta
+      tag: beta
     when:
       event: tag
 
+---
+kind: pipeline
+type: docker
+name: docker-linux-amd64
+depends_on:
+  - main
+trigger:
```

Comment:
can we not just use 

``` 
trigger:
  branch:
  - master
  event:
  - tag
```

instead of the triggers?

Reply:
I'm not sure. Will need to test this.

Reply:
Looks like 
```yaml
trigger:
  event:
    - tag
 ```
works as expected.